featuretools/wrappers/sklearn.py

```diff
@@ -0,0 +1,211 @@
+                between entities. List items are a tuple with the format
+                (parent entity id, parent variable, child entity id, child
+                variable).
+
+            entityset (EntitySet): An already initialized entityset. Required
+                if entities and relationships are not defined.
+
+            target_entity (str): Entity id of entity on which to make
+                predictions.
+
+            agg_primitives (list[str or AggregationPrimitive], optional): List
+                of Aggregation Feature types to apply.
+
+                    Default: ["sum", "std", "max", "skew", "min", "mean",
+                              "count", "percent_true", "n_unique", "mode"]
+
+            trans_primitives (list[str or TransformPrimitive], optional):
+                List of Transform Feature functions to apply.
+
+                    Default: ["day", "year", "month", "weekday", "haversine",
+                              "num_words", "num_characters"]
+
+            allowed_paths (list[list[str]]): Allowed entity paths on which to
+                make features.
+
+            max_depth (int) : Maximum allowed depth of features.
+
+            ignore_entities (list[str], optional): List of entities to
+                blacklist when creating features.
+
+            ignore_variables (dict[str -> list[str]], optional): List of
+                specific variables within each entity to blacklist when
+                creating features.
+
+            seed_features (list[:class:`.PrimitiveBase`]): List of manually
+                defined features to use.
+
+            drop_contains (list[str], optional): Drop features
+                that contains these strings in name.
+
+            drop_exact (list[str], optional): Drop features that
+                exactly match these strings in name.
+
+            where_primitives (list[str or PrimitiveBase], optional):
+                List of Primitives names (or types) to apply with where
+                clauses.
+
+                    Default:
+
+                        ["count"]
+
+            max_features (int, optional) : Cap the number of generated features
+                    to this number. If -1, no limit.
+
+            profile (bool, optional): Enables profiling if True.
+
+        Example:
+            .. code-block:: python
+                import featuretools as ft
+                from sklearn.preprocessing import StandardScaler
+
+                # Get examle data
+                es = ft.demo.load_mock_customer(return_entityset=True)
+                df = es['customers'].df
+
+                # Build dataset
+                pipeline = Pipeline(steps=[
+                    ('ft', dfsTransfomer(entityset=es,
+                                         target_entity="customers",
+                                         max_features=2)),
+                    ('sc', StandardScaler()),
```

Comment:
can we add an classifier to this example since most people will want to use this wrapper to train a model? you can just make random labels to train it one like you do in the tests

then show calling `pipeline.predict()` at the end